elements/lisk-dpos/src/delegates_info.ts

```diff
@@ -157,10 +158,26 @@ export class DelegatesInfo {
 				block.height + 1,
 				stateStore,
 			);
+
+			// Load 3 rounds of headers to calculate random seeds
```

Comment:
wan't it 2 rounds?

Reply:
Yes last two round but to validate last round seeds we need second last round as well.